### PR TITLE
Update nodeJS final_repo_dirs: gcloud-node -> google-cloud-node

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

File: gapic/api/artman_trace.yaml [hunk 1]
```diff
@@ -18,6 +18,6 @@ python:
 ruby:
   final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-trace
 nodejs:
-  final_repo_dir: ${REPOROOT}/artman/output/gcloud-node-trace
+  final_repo_dir: ${REPOROOT}/google-cloud-node-trace/packages/trace
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_trace.yaml]: Why does this one follow a different pattern?

Comment (jmuk) [inline gapic/api/artman_trace.yaml]: oops, that was a silly mistake. thanks for catching this.

**Comment by jmuk** [comment]
@swcloud, can you take a look?

**Comment by geigerj** [comment]
LGTM

**Pull request merged**